tests/console/sqlite3.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Test sqlite3 package
+# Maintainer: Tina Müller <tina.mueller@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use Data::Dumper;
+
+
+sub run {
+    select_console 'root-console';
```

Comment:
Since you're not peforming any screen based testing, I guess this test could benefit from using  `$self->select_serial_terminal` which is probably a bit faster.

Reply:
Ah, I didn't know that. Will have a look!

Reply:
working, thanks!